webservice/src/main/resources/app/scripts/components/TopologyGraphComponent.jsx

```diff
@@ -768,7 +768,7 @@ export default class TopologyGraphComponent extends Component {
           const levelTemp = level > 0 ? level+1 : 1;
           nestedFields(fieldObj[key], levelTemp);
         } else {
-          list.push(`<li style='padding-left : ${level > 0 ? (10*level) : level}px' class=${level > 0 ? 'demo': ''}> ${fieldObj[key]} <span class='output-type'> ${key}</span> </li>`);
+          list.push(`<li style='padding-left : ${level > 0 ? (10*level) : level}px' class=${level > 0 ? 'demo': ''}> ${key} <span class='output-type'> ${fieldObj[key]}</span> </li>`);
```

Comment:
We want to keep the styling of the value and field name as it currently is, just replace the position.